content/docs/dvclive/ml-frameworks/pytorch-lightning.md

```diff
@@ -5,33 +5,54 @@ DVCLive allows you to add experiment tracking capabilities to your
 
 ## Usage
 
-Pass the
+<p align='center'>
+  <a href="https://colab.research.google.com/github/iterative/dvclive/blob/main/examples/DVCLive-PyTorch-Lightning.ipynb">
+    <img src="https://colab.research.google.com/assets/colab-badge.svg" />
+  </a>
+</p>
+
+If you pass the
 [`DVCLiveLogger`](https://github.com/iterative/dvclive/blob/main/src/dvclive/lightning.py)
 to your
-[`Trainer`](https://pytorch-lightning.readthedocs.io/en/latest/common/trainer.html):
+[`Trainer`](https://lightning.ai/docs/pytorch/stable/common/trainer.html),
+DVCLive will automatically log the
+[metrics](https://lightning.ai/docs/pytorch/stable/visualize/logging_basic.html#track-metrics)
+and
+[parameters](https://lightning.ai/docs/pytorch/stable/common/lightning_module.html#save-hyperparameters)
+tracked in your
+[`LightningModule`](https://pytorch-lightning.readthedocs.io/en/2.0.1/common/lightning_module.html).
 
 ```python
+import lightning.pytorch as pl
 from dvclive.lightning import DVCLiveLogger
 
 ...
+class LitModule(pl.LightningModule):
+    def __init__(self, layer_1_dim=128, learning_rate=1e-2):
+        super().__init__()
+        # layer_1_dim and learning_rate will be logged by DVCLive
+        self.save_hyperparameters()
+
+    def training_step(self, batch, batch_idx):
+        metric = ...
+        # some_metric will be logged by DVCLive
+        self.log("some_metric", metric)
```

Comment:
```suggestion
        self.log("train_metric", metric, on_step=False, on_epoch=True)
```

Maybe we should include an example like this and explain how it will get logged to make it more concrete than output format section below?

Reply:
I see the point of the `train_metric` but I am unsure about the other args.
Why do we want to show those details? Feels like duplicate lightning docs

Reply:
The idea was to show how we parse it into separate `step`/`epoch` folders.

This whole section duplicates a lot of lightning docs, but I think it's more user-friendly this way.

Is your concern about maintenance? Or that it makes the example look complicated? Something else?

Reply:
@daavoo ping